canonical-web-and-design/vanilla-framework
# Update vanilla dep to v2.0.1

## Done

Updated the docs to v2.0.1

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check the docs pages in-browser at different viewports
